pattern-library/sass/patterns/_grid.scss

```diff
@@ -14,276 +14,180 @@
 // #GALLERY - CANNED + RESPONSIVE
 // #DEBUG
 
-
 // ----------------------------
-// #SETTINGS
+// #UTILITY
 // ----------------------------
-// grid - configuration (susy-based)
-
-// set RTL/LTR specific variables
-$grid-direction-default: ltr;
-$grid-direction-reversed: rtl;
-
-@if $layout-direction == rtl {
-    $grid-direction-default: rtl;
-    $grid-direction-reversed: ltr;
+// Utility functions/mixins designed for Susy API-compatibility.
+@function cols($cols) {
+    @return $cols / $grid-column-count * 100%;
 }
 
-// susy-based configuration (see http://susydocs.oddbird.net/en/latest/settings/)
-$susy: (
-    use-custom: (
-        background-options: false,
-        box-sizing: false,
-        breakpoint: true,
-        clearfix: true,
-        rem: true
-    ),
-    debug: (
-        color: $grid-debug-color,
-    ),
-    math: fluid,
-    output: float
-);
-
-// LAYOUT: set default susy settings
-// (gutters are handled outside of elements - by default)
-$susy-default: (
-    flow: $grid-direction-default,
-    column-width: $grid-column-width,
-    columns: $grid-column-count,
-    container-position: $grid-container-position,
-    container: $grid-max-width,
-    global-box-sizing: border-box,
-    gutters: $grid-gutter-width,
-    gutter-position: $grid-gutter-position
-);
-
-// LAYOUT: set susy settings especially for manually set grids in the DOM
-// (gutters are handled inside of elements - by default)
-$susy-manual: layout($susy-default $grid-manual-gutter-position);
-
-// LAYOUT: set debug settings
-$susy-debug: layout($susy-default show);
-
-
-// ----------------------------
-// #GLOBALS
-// ----------------------------
-// set basic grid settings for app
-@include layout($susy-default);
+@mixin span($cols, $margin:'') {
```

Comment:
Usage of `span` comes from Susy; their `span` function is actually what was making compilation super slow, I think. I decided to keep the name (and keep it mostly API-compatible) so that migration to this new version will be easier.

e.g., there's about 30-40 places in edx-platform using `span` right now, many of them will just keep working with this new version of `span` (some of them will need changes to the arguments passed).

<details>
```
wheatley > ~ > code > edx-platform > master > $ > git grep "span(" *.scss
cms/static/sass/elements-v2/_header.scss:    @include span(12);
cms/static/sass/elements-v2/_header.scss:    @include span(7);
cms/static/sass/elements-v2/_header.scss:    @include span(4 last);
cms/static/sass/elements-v2/_sock.scss:    @include span(12);
cms/static/sass/elements-v2/_sock.scss:      @include span(8);
cms/static/sass/elements-v2/_sock.scss:      @include span(4);
common/static/sass/bourbon/css3/_columns.scss:@mixin column-span($arg: none) {
common/static/sass/neat/functions/_private.scss:@function container-span($span: $span) {
common/static/sass/neat/grid/_span-columns.scss:  $container-columns: container-span($span);
lms/static/certificates/sass/_components.scss:        width: span(10);
lms/static/certificates/sass/_layouts.scss:        @include span(6 first);
lms/static/certificates/sass/_layouts.scss:        @include span(6 last);
lms/static/certificates/sass/_layouts.scss:            width: span(12);
lms/static/certificates/sass/_layouts.scss:                width: span(10);
lms/static/certificates/sass/_layouts.scss:           width: span(12);
lms/static/certificates/sass/_layouts.scss:               width: span(8);
lms/static/certificates/sass/_layouts.scss:            width: span(6);
lms/static/certificates/sass/_layouts.scss:                width: span(3);
lms/static/certificates/sass/_layouts.scss:                width: span(2);
lms/static/certificates/sass/_layouts.scss:                width: span(2);
lms/static/certificates/sass/_layouts.scss:            width: span(12);
lms/static/certificates/sass/_layouts.scss:                width: span(4);
lms/static/certificates/sass/_layouts.scss:                width: span(3);
lms/static/certificates/sass/_layouts.scss:                width: span(3);
lms/static/certificates/sass/_print.scss:            width: span(8);
lms/static/certificates/sass/_print.scss:            @include span(12 last);
lms/static/certificates/sass/_views.scss:      @include span(9 of 12 first);
lms/static/certificates/sass/_views.scss:      @include span(3 of 12 last);
lms/static/sass/elements/_program-card.scss:  @include span(12);
lms/static/sass/elements/_program-card.scss:    @include span(6);
lms/static/sass/elements/_program-card.scss:    @include span(6);
lms/static/sass/elements/_program-card.scss:    @include span(6);
lms/static/sass/shared-v2/_base.scss:    width: span(12);
lms/static/sass/shared-v2/_footer.scss:            @include span(12);
lms/static/sass/shared-v2/_footer.scss:                @include span(8);
lms/static/sass/shared-v2/_footer.scss:                    @include span(9);
lms/static/sass/shared-v2/_footer.scss:            @include span(4);
lms/static/sass/shared-v2/_footer.scss:            @include span(12);
lms/static/sass/shared-v2/_footer.scss:                @include span(4);
lms/static/sass/shared-v2/_modal.scss:    @include span(5);
lms/static/sass/views/_program-list.scss:  @include span(12);
lms/static/sass/views/_program-list.scss:    @include span(9);
lms/static/sass/views/_program-list.scss:  @include span(12);
lms/static/sass/views/_program-list.scss:    @include span(3);
</details>